app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
+
+  private :update_comments_counter
+end
```

Comment:
- [ ] [OPTIONAL] Just as I have suggested earlier, I would like toto suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new post has been saved to the database, then make the update_comments_counter method private.

Reply:
Thanks for the suggestion. I will study it a bit and try to implement in the next project's phase.